GitUI/CommandsDialogs/RepoHosting/ForkAndCloneForm.Designer.cs

```diff
@@ -507,36 +561,19 @@ private void InitializeComponent()
             this.flowLayoutPanel1.Controls.Add(this.cloneBtn);
             this.flowLayoutPanel1.Dock = System.Windows.Forms.DockStyle.Fill;
             this.flowLayoutPanel1.FlowDirection = System.Windows.Forms.FlowDirection.RightToLeft;
-            this.flowLayoutPanel1.Location = new System.Drawing.Point(3, 515);
+            this.flowLayoutPanel1.Location = new System.Drawing.Point(4, 644);
+            this.flowLayoutPanel1.Margin = new System.Windows.Forms.Padding(4, 4, 4, 4);
             this.flowLayoutPanel1.Name = "flowLayoutPanel1";
-            this.flowLayoutPanel1.Size = new System.Drawing.Size(738, 34);
+            this.flowLayoutPanel1.Size = new System.Drawing.Size(922, 42);
             this.flowLayoutPanel1.TabIndex = 24;
             // 
-            // ProtocolLabel
-            // 
-            this.ProtocolLabel.AutoSize = true;
-            this.ProtocolLabel.Location = new System.Drawing.Point(418, 36);
-            this.ProtocolLabel.Name = "ProtocolLabel";
-            this.ProtocolLabel.Size = new System.Drawing.Size(49, 13);
-            this.ProtocolLabel.TabIndex = 25;
-            this.ProtocolLabel.Text = "Protocol:";
-            // 
-            // ProtocolDropdownList
-            // 
-            this.ProtocolDropdownList.DropDownStyle = System.Windows.Forms.ComboBoxStyle.DropDownList;
-            this.ProtocolDropdownList.FormattingEnabled = true;
-            this.ProtocolDropdownList.Location = new System.Drawing.Point(470, 32);
-            this.ProtocolDropdownList.Name = "ProtocolDropdownList";
-            this.ProtocolDropdownList.Size = new System.Drawing.Size(121, 21);
-            this.ProtocolDropdownList.TabIndex = 2;
-            this.ProtocolDropdownList.SelectedIndexChanged += new System.EventHandler(this.ProtocolSelectionChanged);
-            // 
             // ForkAndCloneForm
             // 
-            this.AutoScaleDimensions = new System.Drawing.SizeF(96F, 96F);
+            this.AutoScaleDimensions = new System.Drawing.SizeF(120F, 120F);
```

Comment:
The design must be done at 100% scale / 96dpi. The latest VS2017 supports design at 100% scaling
![image](https://user-images.githubusercontent.com/4403806/53675120-789a4980-3ce7-11e9-8639-cf81fea53e09.png)

You can start it manually too with
```
"C:\Program Files (x86)\Microsoft Visual Studio\2017\Enterprise\Common7\IDE\devenv.exe" /noScale "<path>\GitExtensions.sln"
```